monad-bayes.cabal

```diff
@@ -1,13 +1,13 @@
 cabal-version:      2.0
 name:               monad-bayes
-version:            0.1.1.0
+version:            0.1.1.2
 license:            MIT
 license-file:       LICENSE.md
 copyright:          2015-2020 Adam Scibior
 maintainer:         dominic.steinitz@tweag.io
 author:             Adam Scibior <adscib@gmail.com>
 stability:          experimental
-tested-with:        GHC ==8.4.4 || ==8.6.5 || ==8.8.1
+tested-with:        GHC ==9.2.2
```

Comment:
We probably ought to do something about this: we should test for 3 versions of ghc I think.

Reply:
https://github.com/tweag/monad-bayes/issues/147